datastore/google/cloud/datastore/client.py

```diff
@@ -15,23 +15,24 @@
 
 import os
 
-from google.cloud.proto.datastore.v1 import datastore_pb2 as _datastore_pb2
-
 from google.cloud._helpers import _LocalStack
-from google.cloud._helpers import (
-    _determine_default_project as _base_default_project)
+from google.cloud._helpers import (_determine_default_project as
+                                   _default_project)
+
```

Comment:
This empty line doesn't belong IIUC (it's pretty irrelevant, but I'm pointing it out for some reason)